### PR TITLE
Home and about

### PR DESCRIPTION
**Pull request by bengigo**
Changes made in this branch: 

- Create 2 pages of a website: Home and About.
- Create both mobile and desktop designs
- Create a mobile menu with active Home and About page links
- Create a dynamic speakers section with JS
- Create the README file of the project
- Prepare and include a presentation video in the README file

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            </ul>
+            <nav class="desktop-nav desktop">
+                <div class="logo-container">
+                    <a href="index.html"><img class="nav-logo-img" src="images/gastronomy-events-logo.png"
+                            alt="logo"></a>
+                    <p class="nav-logo-text">GASTRONOMY <br> EVENTS</p>
+                </div>
+
+                <ul class="desktop-nav-list">
+                    <li class="desktop-links"><a id="about-link" class="desktop-link-titles" href="about.html">About</a>
```
Comment (prolajumokeoni) [inline about.html]: - [x] Kindly change this about.html link to index.html so users can easily navigate to the index.html page from the about 👍🏿

File: about.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+    <title>Gastronomy Retreat</title>
+</head>
+
+<body>
```
Comment (prolajumokeoni) [inline about.html:13]: - [x] Kindly fix horizontal scroll on the about page currently there is a horizontal scroll, please check the widths, padding and margins used to fix this issue, see video below 👍🏿


https://user-images.githubusercontent.com/69638013/169292534-7b496eaa-a164-46b5-abf6-eb8cef095675.mp4

File: index.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+
+            <div class="speaker"></div>
+        </div>
+
+        <!-- <button class="speaker-btn more">More <img class="down" src="images/down.png" alt=""></button>
+
+            <button class="speaker-btn less">Less <img class="down" src="images/up.png" alt=""></button> -->
```
Comment (prolajumokeoni) [inline index.html]: - [x] Kindly delete commented code from your project, you can leave comments about code but not commented code 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @bengigo 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @bengigo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**